examples/python/cli/cli.py

```diff
@@ -214,6 +217,14 @@ def wait_for_payment(sdk: Sdk, destination: str):
             break
         time.sleep(1)
 
+def get_info(params):
+    sdk = Sdk(params.network)
+    res = sdk.instance.get_info()
+    print(json.dumps({
+        "walletInfo": res.wallet_info.__dict__,
+        "blockchainInfo": res.blockchain_info.__dict__,
+    }, indent=2))
```

Comment:
Having issues testing this with assets, this is a bit simpler
```suggestion
    print(json.dumps(res, default=lambda x: x.__dict__, indent=2))
```